ruby/private/bundle/create_bundle_build_file.rb

```diff
@@ -59,7 +59,11 @@
 ALL_GEMS
 
 GEM_PATH = ->(ruby_version, gem_name, gem_version) do
-  "lib/ruby/#{ruby_version}/gems/#{gem_name}-#{gem_version}"
+  Dir.glob("lib/ruby/#{ruby_version}/gems/#{gem_name}-#{gem_version}*").first
```

Comment:
The asterisk is a pretty subtle change. Could you clarify why we are using `*` and why only grab the first item if it's more than one?

Reply:
Oh, I was about to push this commit to explain about this point.

```ruby
# For ordinary gems, this path is like 'lib/ruby/3.0.0/gems/rspec-3.10.0'.
# For gems with native extension installed via prebuilt packages, the last part of this path can
# contain an OS-specific suffix like 'grpc-1.38.0-universal-darwin' or 'grpc-1.38.0-x86_64-linux'
# instead of 'grpc-1.38.0'.
# Since OS platform is unlikely to change between Bazel builds on the same machine, `#{gem_name}-#{gem_version}*`
# would be sufficient to narrow down matches to at most one.
GEM_PATH = ->(ruby_version, gem_name, gem_version) do
  Dir.glob("lib/ruby/#{ruby_version}/gems/#{gem_name}-#{gem_version}*").first
end

# For ordinary gems, this path is like 'lib/ruby/3.0.0/specifications/rspec-3.10.0.gemspec'.
# For gems with native extension installed via prebuilt packages, the last part of this path can
# contain an OS-specific suffix like 'grpc-1.38.0-universal-darwin.gemspec' or 'grpc-1.38.0-x86_64-linux.gemspec'
# instead of 'grpc-1.38.0.gemspec'.
# Since OS platform is unlikely to change between Bazel builds on the same machine,
# `#{gem_name}-#{gem_version}*.gemspec` would be sufficient to narrow down matches to at most one.
SPEC_PATH = ->(ruby_version, gem_name, gem_version) do
  Dir.glob("lib/ruby/#{ruby_version}/specifications/#{gem_name}-#{gem_version}*.gemspec").first
end
```

Reply:
Combining `Dir.glob("...").grep(//)` and `RbConfig['arch']` or something like that could more precisely narrow down the candidate paths. But since I am not sure how many platform variant suffixes exist in the wild and `#{gem_name}-#{gem_version}*` seems to be enough to identify a single directory/path here, I chose the latter (lazy but not too permissive) approach.

Reply:
Feel free to submit another PR with the update, and I'll merge it

Reply:
Nevermind, I just did. Thanks again!